### ISSUE TITLE
After the step 5

### ISSUE THREAD

**Issue by daeyeon** [galpeter/demo-c2c#1]
After step 5, `$ iotjs appRegister.js`, I received the following logs.

```
Update oauth response:
{ clientName: 'Demo client 001',
  scope:
   [ 'r:installedapps:*',
     'r:locations:*',
     'r:schedules',
     'r:devices:$',
     'x:devices:*',
     'l:devices',
     'i:deviceprofiles',
     'w:devices:$',
     'w:schedules',
     'r:devices:*',
     'w:devices:*',
     'x:devices:$' ],
  redirectUris: [] }

Everything should be registered!
Please check the Android App
```

And, result_appdeviceprofile.json and the result_appprofile.json are created as the result. (result_appinstallconfig.json mentioned in step 9 isn't created.)

However, In step 6, I can't find the smartapp (connector) I installed in both the new and the old Android application. It's not found in smartthing.com, but, once I re-request the same device profile and the app, its responses the smartapp is already registered.

Can you answer how to check registered smart apps in the Android apps, or else, the web sites?

**Comment by galpeter** [comment]
The `result_appinstallconfig.json` file be created when the application is installed via the Android app.

Based on my experiments the registered applications can not be viewed on any web site just with the old Android App (if I remember correctly under 'my apps' entry).

**Comment by daeyeon** [comment]
Thanks for the clarification. I also confirmed it in the classic app.

I also got the following mail about this from SmartThings team.

> Based on the response, it looks like the app is successfully registered. I am assuming you are using the new SmartThings app (which was previously called Samsung Connect)?
If so, there's a slight discrepancy between app registration methods. 
When you register an app using the command line, it will show up in the legacy SmartThings app (called SmartThings Classic in Play Store). But it won't be available in the new SmartThings app or the Dev. Workspace Web console. 
We are currently working on synchronizing all of it, so that app registered with either method shows up in both SmartThings app as well as the web console.
